tests/unit/user/payment.test.ts

```diff
@@ -5,22 +5,27 @@ import { connect } from '../../../src/database/db';
 import { Payments } from "../../../src/database/models/Payments";
 import UserController from "../../../src/controllers/UserController";
 import { User } from "../../../src/database/models/User";
+import Status from "../../../src/enum/Status"
+
 
 const expect = chai.expect;
 
+
 describe ("Payment Tests", ()=>{
     let paymentController: PaymentController;
     let userController : UserController;
-
+ 
     before (async ()=> {
         await connect();
-        await Payments.deleteMany({});
+        await Payments.deleteMany({}); 
         paymentController = new PaymentController();
         userController = new UserController();
     });
 
     it ("PaymentController::createPayment successfully upload user in DB", async () =>{
```

Comment:
the description of this test is sort of misleading, the focus isn't to see if UserController works as intended, you are testing to see if PaymentsController is working